src/gt4py/backend/templates/dawn_computation.src.in

```diff
@@ -93,7 +107,7 @@ data_store_t make_data_store(const BufferInfo& bi,
     double* ptr = static_cast<double*>(bi.ptr);
     for (int i = 0; i < 3; ++i) {
         strides[i] = bi.strides[i] / sizeof(double);
-        ptr += strides[i] * origin[i];
+        ptr += strides[i] * (origin[i] - halos_[i*2]);
```

Comment:
Same question as before: Do we really need to set halos in the `data_store` because the grid generated by Dawn requires them? In the gridtools backends we completely avoid them

Reply:
I will experiment with the same method as the gridtools backends.

Reply:
Setting the compute domain halos equal to the origin and removing the pointer increments allows the halos to be avoided as in the gridtools backends. Thank you for the suggestion.